modules/ding_user_form/ding_user_form.info

```diff
@@ -0,0 +1,11 @@
+name = Ding user form
+description = Defines the user login form (optional)
+package = Ding!
+core = 7.x
+dependencies[] = ding_provider
+dependencies[] = ding_popup
+dependencies[] = ding_entity
+dependencies[] = profile2
```

Comment:
If this module is responsible for providing a user login form, then does it really depend on `profile2` and `ding_entity`? That does not seem right. I would on the other hand expect a dependency on `ding_user`.

Reply:
Fixed.